nemo/collections/asr/parts/jasper.py

```diff
@@ -309,21 +309,22 @@ def __init__(
                 nn.Linear(channels, channels // reduction_ratio, bias=False),
                 activation,
                 nn.Linear(channels // reduction_ratio, channels, bias=False),
-            )
+            ).float()
```

Comment:
Do you need this .float()?

Reply:
If AMP is enabled, the parameters of Linear can be initialized as float16. 
This is not desirable as we use float32 in block.

Reply:
Can you use the torch amp decorator and set enabled to False instead ?

Reply:
> If AMP is enabled, the parameters of Linear can be initialized as float16.

Are you sure about this? Doesn't amp autocast them as needed?

Reply:
After going through the documentation, I think you're correct. Explicit conversion is not needed.